sql/core/src/main/scala/org/apache/spark/sql/streaming/StreamingQueryListener.scala

```diff
@@ -140,7 +140,8 @@ object StreamingQueryListener {
   @Evolving
   class QueryIdleEvent private[sql](
       val id: UUID,
-      val runId: UUID) extends Event
+      val runId: UUID,
+      val timestamp: String) extends Event
```

Comment:
It seems better to document what the timestamp is.

Reply:
For example, the python one looks good.

Reply:
I'll bring the parameters in the classdoc for the same of python side. Thanks for the input!